src/components/tooltip/types.ts

```diff
@@ -0,0 +1,58 @@
+import { Position } from '@reach/tooltip'
+import { ReactElement, ReactNode } from 'react'
+export interface TooltipProps {
+  /**
+   * Element that, when hovered, will display the tooltip.
+   */
+  children: ReactElement
+
+  /**
+   * Plain text, or `<ReactElement />`, to render in the tooltip.
+   */
+  label: ReactNode
+
+  /**
+   * Optional alternate content for screen readers.
+   * Useful if the label prop is not suitable for screen reader announcement,
+   * for example for icon-only tooltips.
+   */
+  ariaLabel?: string
```

Comment:
This will _likely_ be useful if we have tooltips with more complex or less "text"-y content. For now it's not used, but figured it was worth including as it seems like a nice thing to be able to reach for when the situation comes up.

Reply:
Agreed. I think even for the 'interactive' and 'video' badges now, a more descriptive label could be useful.